src/System.IO.FileSystem/src/project.json

```diff
@@ -23,14 +23,14 @@
         "dotnet5.4"
```

Comment:
It does, this guy should be `dotnet5.8` and the one in line 46 should be `dotnet5.4` after your changes.